src/components/Loader/Loader.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./Loader.scss";
+import { Hourglass } from "react-loader-spinner";
```

Comment:
use [this](https://mhnpd.github.io/react-loader-spinner/docs/components/oval) instead

Reply:
done